style.css

```diff
@@ -0,0 +1,212 @@
+body {
+	margin: 0;
+	font-family:'Montserrat-Regular';
+	font-style: normal;
+	box-sizing: border-box;
+	background: #E5E5E5;
+}
+
+.header {
+	display: flex;
+	align-items: center;
+}
+.header__logo {
+	display: flex;
+    padding-top: 45px;
+    padding-left: 54px;
+    margin-right: 259px;
+    margin-bottom: 52px;
+	font-family: 'Montserrat-Medium';
+	font-weight: 500;
+	font-size: 30px;
+	text-align: center;
+	line-height: 37px;
+	color: #FFFFFF;
+	cursor: pointer;
+}
+.header__logo-picture {
+	width: 100px;
+	height: 37px;
+}
+.header__lang {
+	margin-right: 1000px;
+	margin-bottom: 10px;
+	cursor: pointer;
+}
+.header__lang-ru {
+	width: 31px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633355)